utils/installer/dependencies for LunarVim (win x64).bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+echo Installing Chocolatey..
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+
+
+
+echo Installing Vim...
+choco install Vim

```

Comment:
- Why is there no `-y` for vim and neovim?
- Why is [vim](https://community.chocolatey.org/packages/vim) capitalized?